www/front_src/src/Resources/Details/tabs/Graph/index.tsx

```diff
@@ -36,19 +38,33 @@ const useStyles = makeStyles((theme: Theme) => ({
 const GraphTabContent = ({ details }: TabProps): JSX.Element => {
   const classes = useStyles();
 
-  const type = details?.type as string;
-  const equalsService = equals('service');
-  const equalsMetaService = equals('metaservice');
+  const type = details?.type as ResourceType;
+  const equalsService = equals(ResourceType.service);
+  const equalsMetaService = equals(ResourceType.metaservice);
+  const equalsAnomalyDetection = equals(ResourceType.anomalydetection);
 
-  const isService = or(equalsService(type), equalsMetaService(type));
+  const { loadDetails } = useLoadDetails();
+
+  const isService =
+    equalsService(type) ||
+    equalsMetaService(type) ||
+    equalsAnomalyDetection(type);
+
+  const getIsReload = (value: boolean): void => {
```

Comment:
```suggestion
  const getIsReload = (value: boolean): void => {
```

this is not a getter, the function should be named 'reload'